Assets/Scripts/Core/MapEditorSaveSystem.cs

```diff
@@ -6,9 +6,9 @@
 namespace OperationBlackwell.Core {
 	public static class SaveSystem {
 
-		private const string SAVE_EXTENSION = "txt";
+		private const string SAVE_EXTENSION = "json";
 
-		private static readonly string SAVE_FOLDER = Application.dataPath + "/SavesWorld/";
+		private static readonly string SAVE_FOLDER = Application.streamingAssetsPath + "/SavesWorld/";
```

Comment:
Coding style: private variables need to have a trailing "_"

Reply:
For these kinds of constants we don’t do that. However, the isInit variable _does_ need one, so good catch.